src/tsickle.ts

```diff
@@ -2025,12 +2025,16 @@ export function emitWithTsickle(
     }));
     tsickleSourceTransformers.push(
         classDecoratorDownlevelTransformer(typeChecker, tsickleDiagnostics));
+  } else {
+    // In either case, run a transformation that synthesizes all comments for consistency.
```

Comment:
"In either case" here makes me think we do this if either of the above are true, but really only happens if they are both false.  Maybe you mean "in any case"?

Reply:
Reworded.